### ISSUE TITLE
activate sleep timer on start playing

### ISSUE THREAD

**Issue by clausmuus** [PaulWoitaschek/Voice#988]
It should be configuable, that the sleep timer starts on playback start after a longer pause or if the app wars startet. On a schort pause the last startus should be remembered. After every pause the timer should starts from beginning, if it is enabled.

**Comment by bensonzachariah** [comment]
This!
I completely agree with everything you suggested. The only issue I have with voice is that I should manually turn on sleep timer everytime I start listening.

Also it would be better if the sleep timer countdown is displayed on the timer icon itself as opposed to the corner of the cover.

**Comment by stevecopley** [comment]
Yes, agree... Came here to see if anyone else had requested this... I'm constantly forgetting to start the timer and waking up to a finished book.

**Comment by helmo** [comment]
This is almost the same as #742 and would be really nice to have.

**Comment by bjohas** [comment]
Agreed!
